### PR TITLE
update(trad): fr all variable

### PR DESCRIPTION
**Pull request by Sebio1980**
All translation in French it's Done.

### CODE REVIEW

File: theme.qml [hunk 1]
```diff
@@ -269,6 +285,14 @@ FocusScope {
...
         },
         "fr": {
+            global_dimm: "Assombrir",
+            global_bright: "Illuminer",
+            global_Dark: "Sombre",
+            global_Light: "Clair",
+            global_OzoneDark: "OzoneDark",
+            global_SteamOS: "SteamOS",
+            global_mono: "Monochrome",
+            global_color: "Couleur",
```
Comment (TigraTT-Driver) [inline theme.qml:295]:
```suggestion
            settings_collection_accentColor_dimm: "Assombrir",
            settings_collection_accentColor_bright: "Illuminer",
            settings_general_colorScheme_Dark: "Sombre",
            settings_general_colorScheme_Light: "Clair",
            settings_general_colorScheme_OzoneDark: "OzoneDark",
            settings_general_colorScheme_SteamOS: "SteamOS",
            settings_general_logoVariant_mono: "Monochrome",
            settings_general_logoVariant_color: "Couleur",
```
global is for text strings which can appear in many places in the theme. I assume these strings will never be needed outside the settings screen. Therefore I would have adapted the naming scheme to this one

File: theme.qml [hunk 2]
```diff
@@ -177,6 +177,14 @@ FocusScope {
...
     property variant dataText: {
         "en": {
+            global_dimm: "dimm",
+            global_bright: "bright",
+            global_Dark: "Dark",
+            global_Light: "Light",
+            global_OzoneDark: "OzoneDark",
+            global_SteamOS: "SteamOS",
+            global_mono: "mono",
+            global_color: "color",
```
Comment (TigraTT-Driver) [inline theme.qml:187]:
```suggestion
            settings_collection_accentColor_dimm: "dimm",
            settings_collection_accentColor_bright: "bright",
            settings_general_colorScheme_Dark: "Dark",
            settings_general_colorScheme_Light: "Light",
            settings_general_colorScheme_OzoneDark: "OzoneDark",
            settings_general_colorScheme_SteamOS: "SteamOS",
            settings_general_logoVariant_mono: "mono",
            settings_general_logoVariant_color: "color",
```
global is for text strings which can appear in many places in the theme. I assume these strings will never be needed outside the settings screen. Therefore I would have adapted the naming scheme to this one

File: theme.qml [hunk 3]
```diff
@@ -223,6 +231,14 @@ FocusScope {
...
         },
         "de": {
+            global_dimm: "dimm",
+            global_bright: "bright",
+            global_Dark: "Dark",
+            global_Light: "Light",
+            global_OzoneDark: "OzoneDark",
+            global_SteamOS: "SteamOS",
+            global_mono: "mono",
+            global_color: "color",
```
Comment (TigraTT-Driver) [inline theme.qml:241]:
```suggestion
            settings_collection_accentColor_dimm: "Leuchtend",
            settings_collection_accentColor_bright: "Gedämpft",
            settings_general_colorScheme_Dark: "Hell",
            settings_general_colorScheme_Light: "Dunkel",
            settings_general_colorScheme_OzoneDark: "OzoneDark",
            settings_general_colorScheme_SteamOS: "SteamOS",
            settings_general_logoVariant_mono: "Einfarbig",
            settings_general_logoVariant_color: "Bunt",
```
global is for text strings which can appear in many places in the theme. I assume these strings will never be needed outside the settings screen. Therefore I would have adapted the naming scheme to this one

File: theme.qml [hunk 4]
```diff
@@ -309,12 +333,20 @@ FocusScope {
...
         },             
         "pt": {
+            global_dimm: "dimm",
+            global_bright: "bright",
+            global_Dark: "Dark",
+            global_Light: "Light",
+            global_OzoneDark: "OzoneDark",
+            global_SteamOS: "SteamOS",
+            global_mono: "mono",
+            global_color: "color",
```
Comment (TigraTT-Driver) [inline theme.qml:349]:
```suggestion
            settings_collection_accentColor_dimm: "dimm",
            settings_collection_accentColor_bright: "bright",
            settings_general_colorScheme_Dark: "Dark",
            settings_general_colorScheme_Light: "Light",
            settings_general_colorScheme_OzoneDark: "OzoneDark",
            settings_general_colorScheme_SteamOS: "SteamOS",
            settings_general_logoVariant_mono: "mono",
            settings_general_logoVariant_color: "color",
```
global is for text strings which can appear in many places in the theme. I assume these strings will never be needed outside the settings screen. Therefore I would have adapted the naming scheme to this one

File: Settings/Settings.qml [hunk 5]
```diff
@@ -56,9 +56,9 @@ FocusScope {
         Component.onCompleted: {
             [
             [ dataText[lang].settings_general_language,  "lang",  "-WIP-",  "en,de,fr,pt" ],
-            [ dataText[lang].settings_general_colorScheme,  "theme",  "",  "Dark,Light,OzoneDark,SteamOS" ],
+            [ dataText[lang].settings_general_colorScheme,  "theme",  "",  dataText[lang].global_Dark + "," + dataText[lang].global_Light + "," + dataText[lang].global_OzoneDark + "," + dataText[lang].global_SteamOS ],
```
Comment (TigraTT-Driver) [inline Settings/Settings.qml:59]:
```suggestion
            [ dataText[lang].settings_general_colorScheme,  "theme",  "",  dataText[lang].settings_general_colorScheme_Dark + "," + dataText[lang].settings_general_colorScheme_Light + "," + dataText[lang].settings_general_colorScheme_OzoneDark + "," + dataText[lang].settings_general_colorScheme_SteamOS ],
```
see suggested changes theme.qml

File: Settings/Settings.qml [hunk 6]
```diff
@@ -56,9 +56,9 @@ FocusScope {
...
             [ dataText[lang].settings_general_muteSounds,  "mutesfx",  "",  dataText[lang].global_no + "," + dataText[lang].global_yes ],
-            [ dataText[lang].settings_general_logoVariant,  "logoVariant",  "",  "mono,color" ],
+            [ dataText[lang].settings_general_logoVariant,  "logoVariant",  "",  dataText[lang].global_mono + "," + dataText[lang].global_color ],
```
Comment (TigraTT-Driver) [inline Settings/Settings.qml:61]:
```suggestion
            [ dataText[lang].settings_general_logoVariant,  "logoVariant",  "",  dataText[lang].settings_general_logoVariant_mono+ "," + dataText[lang].settings_general_logoVariant_color ],
```
see suggested changes theme.qml

File: Settings/Settings.qml [hunk 7]
```diff
@@ -109,7 +109,7 @@ FocusScope {
         id: collectionSettingsModel
         Component.onCompleted: {
             [
-            [ dataText[lang].settings_collection_accentColor,  "accentColor",  "",  "dimm,bright" ],
+            [ dataText[lang].settings_collection_accentColor,  "accentColor",  "",  dataText[lang].global_dimm + "," + dataText[lang].global_bright ],
```
Comment (TigraTT-Driver) [inline Settings/Settings.qml:112]:
```suggestion
            [ dataText[lang].settings_collection_accentColor,  "accentColor",  "",  dataText[lang].settings_collection_accentColor_dimm + "," + dataText[lang].settings_collection_accentColor_bright ],
```
see suggested changes theme.qml

**Review by TigraTT-Driver — CHANGES_REQUESTED** [review]

File: Menu/Menu.qml [hunk 8]
```diff
@@ -185,7 +185,7 @@ FocusScope {
                                     easing.type: Easing.OutExpo
                                 }
                             }
-                            visible: logoVariant == "color"
+                            visible: logoVariant == 1
```
[the remark this answers is not in the document]
Comment (Sebio1980) [reply Menu/Menu.qml:188]: Yes, i see yesterday night and i change in my local. thanks

File: theme.qml [hunk 9]
```diff
@@ -364,8 +396,26 @@ FocusScope {
 
     // Load settings
     property var lang: api.memory.get('lang') || "en"
-    property var theme: api.memory.get('theme') || "Dark"
-    property var logoVariant : api.memory.get('logoVariant') || "mono"
+    //property var theme: api.memory.get('theme') || "Dark"
```
[the remark this answers is not in the document]
Comment (Sebio1980) [reply theme.qml:399]: ok good

File: theme.qml [hunk 10]
```diff
@@ -364,8 +396,26 @@ FocusScope {
...
+            return "Dark";
+        } else if (api.memory.get('themeIndex') == "1") {
+            return "Light";
+        } else if (api.memory.get('themeIndex') == "2") {
+            return "OzoneDark";
+        } else if (api.memory.get('themeIndex') == "3") {
+            return "SteamOS";
+        }
+    }
+    //property var logoVariant : api.memory.get('logoVariant') || "mono"
```
[the remark this answers is not in the document]
Comment (Sebio1980) [reply theme.qml:411]: ok good

File: theme.qml [hunk 11]
```diff
@@ -396,7 +446,14 @@ FocusScope {
             return false;
         }
     }
-    property var accentColor : api.memory.get('accentColor') || "dimm"
+    //property var accentColor : api.memory.get('accentColor') || "dimm"
```
[the remark this answers is not in the document]
Comment (Sebio1980) [reply theme.qml:449]: ok good

File: theme.qml [hunk 12]
```diff
@@ -364,8 +396,26 @@ FocusScope {
...
+        if (api.memory.get('themeIndex') == "0") {
+            return "Dark";
+        } else if (api.memory.get('themeIndex') == "1") {
+            return "Light";
+        } else if (api.memory.get('themeIndex') == "2") {
+            return "OzoneDark";
+        } else if (api.memory.get('themeIndex') == "3") {
+            return "SteamOS";
+        }
+    }
```
[the remark this answers is not in the document]
Comment (Sebio1980) [reply theme.qml:410]: yes i remember, it's a default value. Nice information.

File: theme.qml [hunk 13]
```diff
@@ -396,7 +446,14 @@ FocusScope {
...
     }
-    property var accentColor : api.memory.get('accentColor') || "dimm"
+    //property var accentColor : api.memory.get('accentColor') || "dimm"
+    property var accentColor : {
+        if (api.memory.get('accentColorIndex') == "0") {
+            return "dimm";
+        } else {
+            return "bright";
+        }
+    }
```
[the remark this answers is not in the document]
Comment (Sebio1980) [reply theme.qml:456]: yes i remember, it's a default value. Nice information.

File: theme.qml [hunk 14]
```diff
@@ -364,8 +396,26 @@ FocusScope {
...
+    //property var logoVariant : api.memory.get('logoVariant') || "mono"
+    property var logoVariant : {
+        if (api.memory.get('logoVariantIndex') == "0") {
+            return "mono";
+        } else {
+            return "color";
+        }
+    }
```
[the remark this answers is not in the document]
Comment (Sebio1980) [reply theme.qml:418]: yes i remember, it's a default value. Nice information.

File: theme.qml (hunk 1 above)
Comment (Sebio1980) [reply theme.qml:295]: ok good information, i share this with my friends in our discord channel #Shinretro-theme :)

File: theme.qml (hunk 2 above)
Comment (Sebio1980) [reply theme.qml:187]: ok good information

File: theme.qml (hunk 3 above)
Comment (Sebio1980) [reply theme.qml:241]: ok good information

File: theme.qml (hunk 4 above)
Comment (Sebio1980) [reply theme.qml:349]: ok good information

File: Settings/Settings.qml (hunk 5 above)
Comment (Sebio1980) [reply Settings/Settings.qml:59]: yes of curse

File: Settings/Settings.qml (hunk 6 above)
Comment (Sebio1980) [reply Settings/Settings.qml:61]: yes of curse

File: Settings/Settings.qml (hunk 7 above)
Comment (Sebio1980) [reply Settings/Settings.qml:112]: yes of curse

**Review by TigraTT-Driver — APPROVED** [review]

File: theme.qml [hunk 15]
```diff
@@ -223,6 +231,14 @@ FocusScope {
             menu_games:"games"
         },
         "de": {
+            settings_collection_accentColor_dimm: "Leuchtend",
+            settings_collection_accentColor_bright: "Gedämpft",
+            settings_general_colorScheme_Dark: "Hell",
+            settings_general_colorScheme_Light: "Dunkel",
```
Comment (TigraTT-Driver) [inline theme.qml:237]:
```suggestion
            settings_general_colorScheme_Dark: "Dunkel",
            settings_general_colorScheme_Light: "Hell",
```
Mistake by me on review

**Review by TigraTT-Driver — CHANGES_REQUESTED** [review]

**Review by TigraTT-Driver — APPROVED** [review]

**Pull request merged**